src/systems/diff_drive/DiffDrive.cc

```diff
@@ -144,6 +188,57 @@ void DiffDrive::Configure(const Entity &_entity,
   this->dataPtr->wheelRadius = _sdf->Get<double>("wheel_radius",
       this->dataPtr->wheelRadius).first;
 
+  // Parse speed limiter parameters.
+  bool hasVelocityLimits     = false;
+  bool hasAccelerationLimits = false;
+  bool hasJerkLimits         = false;
+  double minVel   = 0;
+  double maxVel   = 0;
```

Comment:
Should the max values be initialized to `inf` or `std::numeric_limits<double>::max()`? Otherwise, if a user specifies a `min_velocity` for example and forgets to set `max_velocity`, the limiter will clamp the max velocity to 0.

Reply:
Good point, changed in 8876be62680b442c8049d06471abee7b128c1e69.